### DISCUSSION TITLE
在vite2.x中，很多组件无法使用

### DISCUSSION THREAD

**Discussion by bobfw** [ant-design/ant-design#29325] [Q&A] [answered]
期待兼容vite

**Comment by afc163** [comment, marked as answer]
PR is welcome

**Comment by yunsii** [comment]
好奇兼容 vite 需要做些什么呢？

**Reply by yunsii** [reply]
我已经用了，发现问题并通过配置解决样式依赖的问题之后，应该没什么别的问题了 https://github.com/ant-design/ant-design/discussions/29660
